src/docker-runner/utils.ts

```diff
@@ -30,6 +31,7 @@ export const executeDockerRun = async ({ version }: { version: string }) => {
     '-e DOCKER=1',
     argv.configDir ? `-e LOST_PIXEL_CONFIG_DIR=${argv.configDir}` : '',
     isUpdateModeEnabled ? '-e LOST_PIXEL_MODE=update' : '',
+    isGenerateMetaEnabled ? '-e GENERATE_META=true' : '',
```

Comment:
I'd suggest going with a namespaced env like `LOST_PIXEL_GENERATE_META` to prevent collisions and to stick to our standard.